framework/src/modules/chain/chain.js

```diff
@@ -166,14 +158,68 @@ module.exports = class Chain {
 
 			await this._initModules();
 
+			// Prepare dependency
+			const processorDependencies = {
+				blocksModule: this.blocks,
+				logger: this.logger,
+				constants: this.options.constants,
+				exceptions: this.options.exceptions,
+			};
+
+			// TODO: remove this once we have version 2 genesis block
+			this.processor.register(new BlockProcessorV0(processorDependencies), {
+				matcher: ({ height }) => height >= 0 && height <= 0,
```

Comment:
What is this matcher supposed to match? This is the same as `height == 0` and there is no block height 0 (genesis block is height 1)